UsingSeal.md

```diff
@@ -206,16 +206,78 @@ To decrypt an encrypted object in a Move package, follow these steps:
   - Call `bf_hmac_encryption::decrypt` with the encrypted object, the verified derived keys, and the vector of public keys
   - The function returns an `Option<vector<u8>>`. If decryption fails, the return value will be `None`.
 
-> [!NOTE]
-> On-chain decryption currently works only with HMAC-CTR mode, _not_ AES.
+Using the TypeScript SDK, a transaction to call to the on-chain decryption could look something like this:
+```typescript
+// Get derived keys from key servers
+const derivedKeys = await sealClient.getDerivedKeys({
+  id: encryptedObject.id,
+  txBytes, // Should contain call to seal_approve as discussed in the [decryption](#Decryption) section
+  sessionKey,
+  threshold: encryptedObject.threshold,
+});
 
-The published package IDs are:
+// Parse encrypted object as Move object
+const tx = new Transaction();
+const parsedEncryptedObject = tx.moveCall({
+  target: `${SEAL_PACKAGE_ID}::bf_hmac_encryption::parse_encrypted_object`,
+  arguments: [tx.pure.vector("u8", Array.from(encryptedBytes))],
+});
+
+// Get public keys from key servers
+const publicKeys = await sealClient.getPublicKeys(encryptedObject.services.map((service) => service[0]));
```

Comment:
i think that the flow should be different:
- dapp developer selects key servers -> creates PublicKeys onchain
- users can verify those PublicKeys if they want, and then encrypt their data
- anyone/others can later fetch the keys using the ts-sdk, and triger the onchain decryption